lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IMRUDriver.cs

```diff
@@ -307,6 +317,78 @@ public void OnCompleted()
         {
         }
 
+        private IConfiguration GetMapTaskConfiguration(IActiveContext activeContext, string taskId)
+        {
+            IConfiguration mapSpecificConfig;
+
+            if (!_perMapperConfiguration.TryPop(out mapSpecificConfig))
+            {
+                Exceptions.Throw(
+                    new Exception(string.Format("No per map configuration exist for the active context {0}",
```

Comment:
This indicates a bug in IMRU. Do you want to create something like IMRUBugException sort of class?